app/components/TableWrapper.tsx

```diff
@@ -14,11 +14,13 @@ import TableAdvancedSearchFilterToggle from "@/components/TableAdvancedSearchFil
 import TablePaginationControls from "@/components/TablePaginationControls";
 import { useActiveSwitchFilterCount } from "@/components/TableAdvancedSearchFilterToggle";
 import TableResetFiltersToggle from "@/components/TableResetFiltersToggle";
+import { ZodSchema } from "zod";
 
 interface TableProps<T extends Record<string, unknown>> {
   columns: ColDataCardDef<T>[];
   initialQueryConfig: QueryConfig;
   localStorageKey: string;
+  schema: ZodSchema<T>;
```

Comment:
also—you can see that the type of `T` that we're expecting is `Record<string, unknown>`: that's an object with keys that are all strings. We might also be able to find zod generic type to use here, like `ZodRecord`. I don't know that there would be a real benefit to pursuing that change, but it might reveal itself as we spend more time working on components that consume data from the `useQuery` hook...